tests/fixtures/custom_audiences_permissions_all.json

```diff
@@ -25,7 +25,7 @@
       "deleted": false
     }
   ],
-  "data_type": "tailored_audience_permission",
```

Comment:
@BJapac can you remove the `data_type` field from the golden data? Might make sense to regenerate these given that the API response has likely changed.

Reply:
Got it. Will do.